pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -94,6 +94,10 @@ type ControllerConfigSpec struct {
 
 	// proxy holds the current proxy configuration for the nodes
 	Proxy *configv1.ProxyStatus `json:"proxy"`
+
+	// infra holds the infrastructure details
+	// TODO this makes platform redundant as everything is contained inside Infra.Status
```

Comment:
why is this PR making already existing platform redundant? what do you plan to do to with this TODO?

Reply:
If this will get merged then we should remove platform and just get it from infra. Can you suggest different wording or you object the idea?

Reply:
I'm asking why you are adding infra when you could use the already existing platform

Reply:
they need the whole infra object to read the ovirt platform status - this is a prep PR to inject infra